src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput/index.jsx

```diff
@@ -14,12 +14,15 @@ export const AdjustedGradeInput = () => {
     value,
     onChange,
     hintText,
+    possibleGrade,
   } = useAdjustedGradeInputData();
   return (
     <span>
       <Form.Control
-        type="text"
+        type="number"
```

Comment:
When I tested this, I was still able to enter non-numeric characters.  It could be that this is a Paragon bug.

Reply:
It's strange, I'm sure I tested that 🤔
I was able to use numbers, `.`, and `,` only

Reply:
Maybe it's because of the different Paragon versions. If I remember correctly - I tested it on Quince